spring-integration-hazelcast/src/test/java/org/springframework/integration/hazelcast/leader/LeaderInitiatorTests.java

```diff
@@ -307,5 +306,4 @@ public void publishOnGranted(Object source, Context context, String role) {
 		}
 
 	}
-
```

Comment:
I think there is no meaningful changes in this class from you.
So, better to revert altogether.

For note: I prefer to have every single member in the class (including the last one, even if it is a nested class) to be wrapped with blank lines. Much easier to read the code.